lenskit-eval/src/main/java/org/lenskit/eval/traintest/ExperimentJob.java

```diff
@@ -22,6 +22,7 @@
 
```

Comment:
That's exactly what it is. Separated into separate commits, but that doesn't help in the PR diff. Adding the semaphore alone would have made an already-terrible method even *more* indented.